<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rentingdata</class>
 <widget class="QDialog" name="rentingdata">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>预定租房</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>581</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>功能键</string>
   </property>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAlter">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFirst">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>第一条</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFormer">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>前一条</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLatter">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>后一条</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLast">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>最后一条</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSearch">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>查找</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>110</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>共有：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCount">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>条记录</string>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>791</width>
     <height>291</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>求租户资料</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>求租客户编号</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>房型要求</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>定向价位</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>67</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>备注</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>求租客户姓名</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>90</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>面积要求</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>150</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>意向预期</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>30</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>电话</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>90</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>装修要求</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>516</x>
       <y>150</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>意向房屋编号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCustomerId">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRoomType">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPrice">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTime">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>140</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditArea">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>80</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCustomerName">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCircumstance">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>80</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRoomId">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>140</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>元/月</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>90</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>平米 </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>150</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string> 月</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditRemark">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>187</y>
       <width>551</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPhoneNumber">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>30</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>70</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOrder">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>70</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>预约</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
